specifyweb/frontend/js_src/.eslintrc.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
```

Comment:
While this file is a good example of the default eslint config, we already have an eslint config  so you don't have to create a new one. Take a look at `eslint.config.js`